### ISSUE TITLE
SyntaxError when trying to record a GET request with no body

### ISSUE THREAD

**Issue by mj12albert** [ijpiantanida/talkback#16]
Getting this error when trying to record a get request with no body

```
Error handing request SyntaxError: Unexpected end of JSON input
    at JSON.parse
    at Tape.normalizeBody
    at new Tape
```
I think its from here? https://github.com/ijpiantanida/talkback/blob/master/src/tape.js#L69

`JSON.parse` error when `this.req.body` is `undefined`

**Comment by ijpiantanida** [comment]
Is this from the incoming request, or from a saved tape?

Incoming requests always have at least an empty body (https://github.com/ijpiantanida/talkback/blob/master/src/server.js#L22)

And tapes are expected to always have the `body` property, at least set to an empty string.

**Comment by mj12albert** [comment]
I see, this is from the incoming request. (trying to save a tape)

so it must be something wrong with the data in the response body... 🤔

**Comment by ijpiantanida** [comment]
Hi @mj12albert.
It turns out there was a problem with empty requests after all.

I just published v1.8.1 with a fix.
